Assets/Scripts/Models/Furniture.cs

```diff
@@ -342,6 +342,16 @@ public static Furniture PlaceInstance(Furniture proto, Tile tile)
 
     public void Update(float deltaTime)
     {
+        if (HasPower() == false && PowerConnection.IsPowerConsumer)
```

Comment:
I think this would be a little bit faster for failures if you check `PowerConnection.IsPowerConsumer` first